swagger.json

```diff
@@ -5101,15 +5101,41 @@
                 "label",
                 "value"
             ],
+            "description": "A field of a document. A field can be a string type, number type, or photo type. Between stringValue, numberValue, and photoValue, only one can exist for a single document field depending on the field's type.",
             "properties": {
                 "label": {
                     "type": "string",
                     "description": "Descriptive name of this field.",
                     "example": "Fuel Cost ($)"
                 },
                 "value": {
-                    "description":  "Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
+                    "description":  "DEPRECATED: Please use stringValue, numberValue, or photoValue instead. Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
```

Comment:
I couldn't find any other examples where we do this, but do we have a standardized way to deprecate API fields?